package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -822,94 +1109,208 @@ def timeseries(self):
            w.run()
            timeseries = w.timeseries
 
-        See Also
-        --------
-        :attr:`table` : structured array of the data
         '''
-        return super(WaterBridgeAnalysis, self).timeseries
-
-    def generate_table(self):
-        """Generate a normalised table of the results.
 
-        The table is stored as a :class:`numpy.recarray` in the
-        attribute :attr:`~WaterBridgeAnalysis.table`.
-
-        See Also
-        --------
-        WaterBridgeAnalysis.table
-        """
-        super(WaterBridgeAnalysis, self).generate_table()
-
-    def count_by_type(self):
+        def analysis(current, output):
+            output.extend(current)
+
+        timeseries = []
+        for frame in self._network:
+            new_frame = []
+            self._traverse_water_network(frame, [], analysis_func=analysis, output=new_frame, link_func=self._full_link)
+            timeseries.append(new_frame)
+        self._timeseries = timeseries
+        # after 1.0 will be return the unformated _timeseries
+        return [[self._reformat_hb(hb) for hb in hframe] for hframe in self._timeseries]
+
+    @classmethod
```

Comment:
You have to add a bit more explanation for developers how all of this works. It is clear to yourself right now but (1) anyone reviewing this (like me) and (2) any future developers who want to fix something or extend would like to have a quick description of what you're doing here. What are the data structures you're working with, what needs to be done, what functions are needed and why.

Reply:
I have added more explanation and I hope it is clear. If it is not, I can add more examples.